TRACING-3258 | Document troubleshooting OTEL (#64772)
openshift/openshift-docs: modules/distr-tracing-otel-troubleshoot.adoc

Reviewer: I recommend [removing this file](https://github.com/openshift/openshift-docs/pull/64772/files#r1331727890). It's failing the current build.

        logs:
          level: debug <1>
----
<1> Collector's log level. Possible values are `info`, `warn`, `error` and `debug`. Defaults to `info`.

= Metrics

The collector exposes a set of metrics which indicate how much data collector processed.
The metrics can be retrieved by port forwarding collector pod (`oc port-forward <collector-pod>`) and accessing link:http://localhost:8888/metrics[metrics endpoint].

The following metrics are for spans, however similar metrics are exposed for metrics and logs signals:

* `otelcol_receiver_accepted_spans` -  number of spans successfully pushed into the pipeline.
* `otelcol_receiver_refused_spans` - number of spans that could not be pushed into the pipeline.
* `otelcol_exporter_sent_spans` - number of spans successfully sent to destination.
* `otelcol_exporter_enqueue_failed_spans` - number of spans failed to be added to the sending queue.

// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
.OpenTelemetry Collector custom resource with telemetry service
[source,yaml]
----
  config: |
    service:
      telemetry:
        metrics:
          address: ":8888" <1>
----

> <1> The address on which internal collector metrics are exposed. The operator creates service `<cr-name>-collector-monitoring` that can be used to scrape the metrics endpoint. Defaults to `:8888`.

Reviewer: ```suggestion
<1> The address on which internal collector metrics are exposed. The operator creates a  `<cr-name>-collector-monitoring` service that you can use to scrape the metrics endpoint. Defaults to `:8888`.
```

= Logging exporter

> The logging exporter can be configured to export the collected data to the standard output.

Reviewer: ```suggestion
The logging exporter is configurable for exporting the collected data to the standard output.
```
